### ISSUE TITLE
Cinch 0.8.x requires ansible >=2.3.0.0 because of new syntax in ntp/tasks

### ISSUE THREAD

**Issue by jaypoulz** [RedHatQE/cinch#176]
### Overview
After upgrading to cinch 0.8.x, cinch no longer works past the NTP override block. This is a regression from 0.6.0 that appears to be caused by a patch to add an optional NTP Server override. In my configuration, I am not supplying any overridden NTP servers, so defaults are assumed.

##### Versions Affected
0.8.0-present

##### Failing Configurations
linchpin==1.0.2 (*Note* there is an [issue](https://github.com/CentOS-PaaS-SIG/linchpin/issues/356) provisioning from beaker with version 1.0.3)
ansible==2.2.1.0
cinch>=0.8.0

##### Last Successful State Supporting Linchpin 1.0.2
linchpin==1.0.2
ansible== 2.2.1.0
cinch==0.6.0

##### Current Psuedo-Successful State
linchpin==1.0.3 (*Note* there is an [issue](https://github.com/CentOS-PaaS-SIG/linchpin/issues/356) provisioning from beaker with this version)
ansible>= 2.3.0.0
cinch>=0.8.0

**Note** If you already have a provisioned beaker system with a properly configured inventory, cinch will work properly from this point forward.

### Error Message
```
ERROR! 'name' is not a valid attribute for a Block

The error appears to have been in '/usr/lib/python2.7/site-packages/cinch/roles/ntp/tasks/main.yml': line 7, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: override ntp servers when ntp_servers is defined
  ^ here
```
### Analysis / Root Cause
It appears as though because of the syntax used in the new updated NTP file, cinch requires ansible [>=2.3.0.0](https://github.com/ansible/ansible/commit/fcb033b9e442e7e49921817729169a14ba53d99d). In essence, this means that the cinch >=0.8.0 releases are incompatible with linchpin 1.0.2 (or older).

**Comment by greg-hellings** [comment]
#137 isn't the cause of this. If anything, it's the Ansible version that's been upgraded. It must no longer support name attributes on blocks.

**Comment by jaypoulz** [comment]
@greg-hellings That is correct - I've updated this issue with the most up-to-date information from my investigation. The newer versions of Ansible support name attributes on blocks, but since linchpin 1.0.2 requires an older version of Ansible, it is not compatible with the 0.8.x releases. I have linked the reason why I needed linchpin 1.0.2 in the issue as described above. I have since fixed this issue in linchpin, which should be compatible with cinch (for the purposes of provisioning from beaker) in it's next release.

**Comment by greg-hellings** [comment]
We've been pretty well stuck on pre-release versions of linchpin for a long time through our arc, waiting for them to release 1.0.3. We're definitely not going to try and roll backwards and support older versions. I'll bump the versions up before the next cycle.

**Comment by jaypoulz** [comment]
Sounds good! 👍 I'm just reporting this for documentations sake (in case anyone else encountered this issue). Since I don't need any of the updates past 0.6.0 at the moment, I can revert to the previous releases until linchpin 1.0.4 lands. Thanks!

**Comment by robled** [comment]
Can this be closed out?

**Comment by robled** [comment]
This should be resolved in the event that linchpin and cinch are kept in separate virtualenvs (each with their own version of Ansible), which downstream documentation currently reflects, as well as this PR which was released as part of cinch 0.9.0:

https://github.com/RedHatQE/cinch/pull/192

The current release of cinch can use the latest released version of Ansible, which works fine with the 'ntp' role.
